docs/proposals/proposal-002-run.md

```diff
@@ -0,0 +1,313 @@
+implementation. The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories
+
+<!--
+Detail the things that people will be able to do if this proposal is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+**CNCF project maintainer creates a new benchmark test for their project**
+
+As a project maintainer, I create and run a benchmark test from a separate GitHub repository or from the green-reviews-tooling repository, by following the steps indicated in the Green Reviews documentation, so that the project’s benchmark tests runs in the Green Reviews pipeline and so that I can see the project's sustainability metrics.
+
+
+**Green Reviews maintainer helps to create a new benchmark test for a specific CNCF project**
+
+As a Green Reviews maintainer, I can help a CNCF project maintainers to define the Functional Unit of a project so that the project maintainers can create a benchmark test.
+
+**Project maintainer modifies or removes a benchmark test**
+
+As a project maintainer, I can edit or remove a benchmark test if it is in a repository owned by the CNCF project itself, or otherwise if it’s in the Green Reviews repository by making a pull request with the changes.
+
+
+<!--
+As a Green Review WG reviewer,
+
+As a Green Review WG cluster maintainer, -->
+
+### Notes/Constraints/Caveats (Optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate?
+Think broadly.  For example, consider how this will impact or be impacted
+by scaling to support more CNCF Projects.
+
+How will this affect the benchmark tests, CNCF Project Maintainers, pipeline maintainers, etc?
+-->
+
+As with every design document, there’s a risk that the solution doesn’t cover all cases, especially considering that at Green Reviews we have only had one test case (very appreciated guinea pig 🙂), which is Falco. This proposal needs to at least support that case. When other CNCF projects start using Green Reviews we will learn more and adapt the project as needed.
+
+Another risk is the question of ownership between Green Reviews contributors and CNCF project maintainers. As much as possible, Green Reviews contributors should empower project maintainers to create the benchmark tests. This does not stop Green Reviews maintainers from creating new benchmark tests. However, project maintainers should be encouraged to lead this work. Otherwise, this could add load on Green Reviews maintainers and it is hard to scale. Project maintainers should lead this since they have the expertise about how the project scales and performs under load.
```

Comment:
I would simplify a bit and refer to the investigation issue: 

--- 
Another risk is the question of ownership between Green Reviews contributors and CNCF project maintainers. 

CNCF project maintainers should be allowed to define and maintain their own benchmarks to make sure that the right environment is configured. However if every projects run different benchmarks it would be though to compare results (sustainability metrics) between each other. 

This needs [deeper investigations that will be discussed in a separate proposal](https://github.com/cncf-tags/green-reviews-tooling/issues/103).

Reply:
Yes, I think we are talking about two different potential (and perhaps conflicting) issues here:

- The **scalability** if we get many more CNCF projects interested and there's an expectation of Green Reviews WG to participate in writing the tests or in general helping the CNCF projects getting on board.
- The validity of comparing results if every CNCF project has a different criteria for making these tests.

Perhaps we should mention both, with the link you suggest for the second, @AntonioDiTuri ?

Reply:
Yes definitely we should mention both! I also like the bullet point structure with a little tile per every point, let me propose some edits